patches/chrome-browser-resources-downloads-item.css.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/downloads/item.css b/chrome/browser/resources/downloads/item.css
+index f5c2357c5c986e34e7d1256558036e37ecd2effd..edb2ac5e05bfd1a1be06bdf0c0cff72d2152755e 100644
+--- a/chrome/browser/resources/downloads/item.css
++++ b/chrome/browser/resources/downloads/item.css
+@@ -69,7 +69,7 @@ cr-button {
+ @media (prefers-color-scheme: light) {
+   #content:not(.is-active) {
+     /* GM3 color for "filled cards" */
```

Comment:
oh man, this would be so much easier if they used a CSS variable here :facepalm:

Reply:
Yep, I was very aggravated to see the hard-coded rgb value.

Reply:
Its mad because they actually define all of these

Reply:
brave://theme/colors.css?sets=ui,chrome